### DISCUSSION TITLE
problem uploading my game xtypes (68k size) on tic80

### DISCUSSION THREAD

**Discussion by t00cg** [nesbox/TIC-80#1554] [General]
hi there, 

i have a problem to upload my latest product to tic80.com
http://www.la1n.ch/xtypes/xtypes.tic (148kb)

in my opinon there is everything in it title, autor, cover and so on. the thing is quite big - 86k. size cod.
i stored everything with F7.

my metadatas are:
// title:  xtypes
// author:    la1n
// desc:   shootemup
// script: js

 if i upload it - i got  the error - title, author or cover missing.

could someone help me?
https://tic80.com/play?cart=2102

thanks for any help

**Comment by nesbox** [comment]
Hi, I'll look at it in 2-4 hrs

**Comment by t00cg** [comment]
i created now a 64kb source code version and now i can upload it. 

so the problem is the size of the source code. is this intended or a bug?

thanks for any reply.
